framework/auth/core.py

```diff
@@ -1292,7 +1297,7 @@ def merge_user(self, user):
                 node.save()
 
         # - projects where the user was the creator
-        for node in user.node__created:
+        for node in user.created:
```

Comment:
I removed the created property and all tests passed just fine! Happy to do it. 

Would this be just removing the created property on the user, or the creator field from the node itself?